src/main/docs/guide/junit5.adoc

```diff
@@ -112,6 +112,24 @@ Or the packages:
 
 To ensure that entities can be found during classpath scanning.
 
+=== Transaction semantics
```

Comment:
Could we create an asciidoc snippet and include it in the four files so that we don't have this section duplicated four times.

Reply:
I'll do that.

But bear in mind that pretty much the whole documentation is repeated. Eg, the previous section `Environments, Classpath Scanning etc` is also duplicated four times.